src/styles/index.scss

```diff
@@ -1,3 +1,5 @@
-body {
-  margin: 0;
-}
+@import '/src/styles/blocks/html/body';
+@import '/src/styles/blocks/card';
+@import '/src/styles/blocks/section1';
+@import '/src/styles/blocks/section2';
+@import '/src/styles/blocks/section3';
```

Comment:
The import paths should not include '/src' as SCSS/Sass resolves the paths relative to the file. Use relative paths like './blocks/html/body' instead.